components/service/glean/scripts/sdk_generator.gradle

```diff
@@ -100,6 +90,21 @@ task createBuildDir {
     }
 }
 
+// Configure the Python environments
+envs {
+    bootstrapDirectory = GLEAN_BOOTSTRAP_DIR
+    pipInstallOptions = "--trusted-host pypi.python.org --no-cache-dir"
+
+    // Setup a miniconda environment. conda is used because it works
+    // non-interactively on Windows, unlike the other options
+    conda "Miniconda3", "Miniconda3-${MINICONDA_VERSION}", "64", ["glean_parser==${GLEAN_PARSER_VERSION}"]
+}
+
+tasks.whenTaskAdded { task ->
```

Comment:
Can you add a comment here explaining why this is required?

Reply:
Sure.  I added detail to the already-existing comment above about why JetBrain's suggested solution isn't sufficient.